srcs/client/src/App.jsx

```diff
@@ -0,0 +1,65 @@
+import Navbar from "./components/Navbar";
+import Home from "./Home";
+import Login from "./Login";
+import Register from "./Register";
+import Logout from "./components/logout";
+import TwoFactorLogin from "./components/TwoFactorLogin";
+import "./App.css";
+import { BrowserRouter, Routes, Route, Navigate } from "react-router-dom";
+import { useEffect, useState } from "react";
+
+const App = () => {
+  const [user, setUser] = useState(null);
+  const [loginOrRegister, setAuthState] = useState(null);
+  useEffect(() => {
+    const getUser = () => {
+      fetch("http://127.0.0.1:4200/auth/success", {
```

Comment:
About this address I am not sure about the effect in production as we do not expose api server to the client. We may have to think about it soon.

Reply:
I'm not sure I follow. How would the client make requests to start a new game for example or to get user data ?

But yeah this isn't the cleanest way of doing it and I think the team is going to do something else. This was just to test the flow.

Reply:
In production the website will be served by a web server (nginx for example). As a user, I should not have to access a backend directly to process my queries. I think the proper way to do it in production will be with nginx proxying backend traffic with a specific location (/api for example).